app/src/main/java/ch/epfl/cs311/wanderwave/ui/components/profile/ImagesActions.kt

```diff
@@ -3,25 +3,47 @@ package ch.epfl.cs311.wanderwave.ui.components.profile
 import android.net.Uri
 import androidx.activity.compose.rememberLauncherForActivityResult
 import androidx.activity.result.contract.ActivityResultContracts
-import androidx.compose.foundation.Image
-import androidx.compose.foundation.clickable
+import androidx.compose.foundation.background
 import androidx.compose.foundation.layout.Box
 import androidx.compose.foundation.layout.fillMaxWidth
 import androidx.compose.foundation.layout.padding
 import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
 import androidx.compose.runtime.Composable
 import androidx.compose.runtime.getValue
 import androidx.compose.runtime.mutableStateOf
 import androidx.compose.runtime.remember
 import androidx.compose.runtime.setValue
 import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
 import androidx.compose.ui.platform.testTag
-import androidx.compose.ui.res.painterResource
 import androidx.compose.ui.unit.dp
-import ch.epfl.cs311.wanderwave.R
 import ch.epfl.cs311.wanderwave.model.data.Profile
-import coil.compose.AsyncImage
+import ch.epfl.cs311.wanderwave.ui.theme.ProfileBackground1
+import ch.epfl.cs311.wanderwave.ui.theme.ProfileBackground10
+import ch.epfl.cs311.wanderwave.ui.theme.ProfileBackground11
+import ch.epfl.cs311.wanderwave.ui.theme.ProfileBackground12
+import ch.epfl.cs311.wanderwave.ui.theme.ProfileBackground13
+import ch.epfl.cs311.wanderwave.ui.theme.ProfileBackground14
+import ch.epfl.cs311.wanderwave.ui.theme.ProfileBackground15
+import ch.epfl.cs311.wanderwave.ui.theme.ProfileBackground16
+import ch.epfl.cs311.wanderwave.ui.theme.ProfileBackground17
+import ch.epfl.cs311.wanderwave.ui.theme.ProfileBackground18
+import ch.epfl.cs311.wanderwave.ui.theme.ProfileBackground19
+import ch.epfl.cs311.wanderwave.ui.theme.ProfileBackground2
+import ch.epfl.cs311.wanderwave.ui.theme.ProfileBackground20
+import ch.epfl.cs311.wanderwave.ui.theme.ProfileBackground21
+import ch.epfl.cs311.wanderwave.ui.theme.ProfileBackground3
+import ch.epfl.cs311.wanderwave.ui.theme.ProfileBackground4
+import ch.epfl.cs311.wanderwave.ui.theme.ProfileBackground5
+import ch.epfl.cs311.wanderwave.ui.theme.ProfileBackground6
+import ch.epfl.cs311.wanderwave.ui.theme.ProfileBackground7
+import ch.epfl.cs311.wanderwave.ui.theme.ProfileBackground8
+import ch.epfl.cs311.wanderwave.ui.theme.ProfileBackground9
```

Comment:
I'm nitpicking, but it would be a little bit cleaner to have a way to generate colors than just a list of themes like that